### DISCUSSION TITLE
Why these two links in pkgcommand script are different?

### DISCUSSION THREAD

**Discussion by hahaleyile** [James-Yu/LaTeX-Workshop#3441] [Q&A] [answered]
https://github.com/James-Yu/LaTeX-Workshop/blob/cf9cfa0e9730946ccddbf9934bfa96c51675c508/dev/pkgcommand.py#L23
https://github.com/James-Yu/LaTeX-Workshop/blob/cf9cfa0e9730946ccddbf9934bfa96c51675c508/dev/pkgcommand.py#L39

I found that the auto-completion file obtained from the link below is not the latest. What is the reason?

**Comment by jlelong** [comment, marked as answer]
https://github.com/jlelong/LaTeX-cwl/ is a fork of https://github.com/LaTeXing/LaTeX-cwl, because the latter did not seem maintained anymore. So I decided to fork it to fix typos more easily. 

I have updated my fork and pushed the new auto-completion files. Close by fb4872c2
